ParisBNF/abb/BNFabb226.xml

```diff
@@ -0,0 +1,523 @@
+                                    <dim unit="leaf">1</dim>
+                                    <locus target="#112"/>
+                                    s.l. 1, stab before 1
+                                 </item>
+                                   </list>
+                                 </collation>
+                                <condition key="good">Half of the text column on <locus target="#75vb"/> has been erased.
+                                The leather cover is torn along the spine (as if the
+                                binding would have been too tight for the textblock),
+                                slightly damaged or abraded on the corners and edges of the boards.
+                               </condition>
+                                </supportDesc>
+
+                           <layoutDesc>
+                              <layout columns="2" writtenLines="24">
+                              <ab type="pricking">Pricking is visible.</ab>
+                                  <ab type="ruling">Ruling is visible.</ab>
+                                <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                <ab type="ruling">The upper line is written above the ruling.</ab>
+                                <note>The leaves <locus target="#i #1"/>are neither pricked nor ruled;
+                                last leaf <locus target="#112"/> is ruled and pricked in part.</note>
+                              </layout>
+                           </layoutDesc>
+
+                          </objectDesc>
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script">Late 18th/first half of the 19th century.</seg>
+                              <seg type="ink">Black; red.</seg>
+                              <desc>Careful, regular; by a well-traned scribe (the name of the scribe seems to be
+                              <persName ref="PRS13845HaylaMikael" role="scribe">Ḫāyla Mikāʾel</persName>, see <locus target="#109vb"/>);
+                                described by A. d'Abbadie as <foreign xml:lang="fr">ʻécriture guịlḥʼ</foreign>.</desc>
+                              </handNote>
+
+                                </handDesc>
+
+                                <decoDesc>
+                                  <summary>The legends of the miniatures are written in the upper margin,
+                                    decorated by dotted frames.
+                                  </summary>
+                                    <decoNote type="miniature" xml:id="d1">
+                                         <locus target="#1v"/>
+                                         <desc><ref type="authFile" corresp="AT1086StrikingHead"/> with the legend
+                                         <q><foreign xml:lang="gez">ዘከመ፡ ኰርዕዎ፡ ርእሶ፡ ለእግዚእነ፨</foreign></q>
```

Comment:
legends should be encoded with  `<q xml:lang="gez">`, foreign is used for captions only